Makefile

```diff
@@ -47,3 +47,13 @@ push-gateway:
 
 push-repository-server:
 	docker push $(repositoryServerImageName):$(version)
+
+compose-up:
+ifeq ($(arch),arm)
+	docker-compose -f docker-compose-dev-arm64.yml up
+else
+	docker-compose -f docker-compose-dev.yml up
```

Comment:
Should we use the `profile: full` here?

Reply:
I'll add it. Thanks!  I'm actually using `nerdctl` that doesn't support the `profile` so I didn't need it.